packages/world/ts/config/v2/input.ts

```diff
@@ -91,6 +88,20 @@ export type DeployInput = {
   readonly worldsFile?: string;
   /** Deploy the World as an upgradeable proxy */
   readonly upgradeableWorldImplementation?: boolean;
+  /**
+   * Deploy the World using a custom implementation. This world must implement the same interface as World.sol so that it can initialize core modules, etc.
```

Comment:
```suggestion
   * Deploy the World using a custom implementation. This world must implement the same interface as `World.sol` so that it can initialize core modules, etc.
```